### PR TITLE
If a query is terminated you were unable to use <! on a query name

### PR DESCRIPTION
**Pull request by steveYeah**
This only happened when using Postgres.

This removes the final `;` in a Postgres query to allow the RETURNING id
This is one of the fixes suggested in #26

### CODE REVIEW

**Comment by steveYeah** [comment]
@nackjicholson @honza 
Any thoughts on this? Not a complete an final solution but a step in the right direction I think. Thanks

**Comment by honza** [comment]
Sorry, I haven't had much time to devote to code review. I'll try to get to it soon!

**Comment by steveYeah** [comment]
Thanks @honza, totally understood, no panic 😄

File: anosql/core.py [hunk 1]
```diff
@@ -88,14 +88,16 @@ def parse_sql_entry(db_type, e):
...
     if query == '':
         return None, None
 
-    if sql_type == AUTO_GEN and db_type == 'postgres':
-        query += ' RETURNING id'
-
     if db_type == 'postgres':
+        if sql_type == AUTO_GEN:
+            query = query[:-1] if query[-1] == ';' else query
+            query += ' RETURNING id'
```
Comment (nackjicholson) [inline anosql/core.py]: It's okay to not put the `;` back on right? This shuffle of things looks right to me @steveYeah. Good stuff.

Comment (steveYeah) [reply anosql/core.py]: Yeah, it's not required 👍

**Pull request merged**